### DISCUSSION TITLE
Access to the image name at runtime

### DISCUSSION THREAD

**Discussion by jeanbisutti** [oracle/graal#8488] [Q&A] [answered]
Hi,
Could you please let me know if a GraalVM environment variable or property can provide the image name at runtime?
Thanks!

**Comment by alina-yur** [comment, marked as answer]
Hi @jeanbisutti. You can the image name either via command line option `-o`:

```-o myappname```



Or via the `ImageName` property in `native-image.properties` in  META-INF/native-image/ .
